managed/models/agent_helpers.go

```diff
@@ -73,6 +74,7 @@ func PostgreSQLOptionsFromRequest(params PostgreSQLOptionsParams) *PostgreSQLOpt
 	// PostgreSQL exporter has these parameters but they are not needed for QAN agent.
 	if extendedOptions, ok := params.(PostgreSQLExtendedOptionsParams); ok && extendedOptions != nil {
 		res.AutoDiscoveryLimit = extendedOptions.GetAutoDiscoveryLimit()
+		res.MaxExporterConnections = extendedOptions.GetMaxConnections() // TODO: is it a right place for this?
```

Comment:
Is it?

Reply:
@JiriCtvrtka Do you know why do we need extended options at all?

Reply:
@artemgavrilov I far as I remember we using it for unique properties l. Like property is needed for postgres_exporter but not useful for pg_stat_monitor etc.

Reply:
Ok, thanks. Then it's right place for new parameter